lib/standard/formatter.rb

```diff
@@ -15,6 +18,7 @@ def initialize(*args)
 
     def file_finished(file, offenses)
       uncorrected_offenses = offenses.reject(&:corrected?)
+      @autocorrectable_offenses = uncorrected_offenses.select(&method(:autocorrectable_offense?))
```

Comment:
I used the (possibly) esoteric `method` method to avoid writing a 3-line block.

This line is equivalent to:
```ruby
@autocorrectable_offenses = uncorrected_offenses.select do |offense| 
  autocorrectable_offense?(offense)
end
```

I'm more than willing to make the aforementioned change if someone thinks that this kind of esoteric methods are best avoided.